src/test/java/task40/helpes/LoginPage.java

```diff
@@ -9,17 +9,9 @@
 
 import java.time.Duration;
 
-public class LoginPage {
-    private static final String URL = "https://mail.yandex.com";
-    private final WebDriver driver;
-
-    public LoginPage(WebDriver driver) {
```

Comment:
It was a good practice to have constructor with WebDriver. Please return it back.
The bad practice is when you need additional parameter driver for all your methods in the class due to reason that you don't have constructor.